db/change_cache_test.go

```diff
@@ -1185,3 +1187,114 @@ func readNextFromFeed(feed <-chan (*ChangeEntry), timeout time.Duration) (*Chang
 	}
 
 }
+
+
+// This was added to increase code coverage as part of SG #2633
+// It was artifically derived to pick one arbitrarily chosen code block in the change set mentioned
+// in that issue, to try to exercise that code.  If that code block is commented, it does catch
+// error which demonstrates that the code block is useful.
```

Comment:
I'm not sure which code block you're talking about commenting here - everything in the range over UnusedSequences?